astropy/units/quantity_helper/helpers.py

```diff
@@ -442,9 +402,10 @@ def helper_clip(f, unit1, unit2, unit3):
 
 # ufuncs that do not care about the unit and do not return a Quantity
 # (but rather a boolean, or -1, 0, or +1 for np.sign).
-onearg_test_ufuncs = (np.isfinite, np.isinf, np.isnan, np.sign, np.signbit)
-for ufunc in onearg_test_ufuncs:
-    UFUNC_HELPERS[ufunc] = helper_onearg_test
+UFUNC_HELPERS |= dict.fromkeys(
+    (np.isfinite, np.isinf, np.isnan, np.sign, np.signbit),
+    lambda f, unit: ([None], None),
+)
```

Comment:
This is a nice simplification, but I'd love some explanatory comments on the form of the function.